data/PhyloData_ex/phylolikelihood_hmm_input.csv

```diff
@@ -0,0 +1,2 @@
+names flexbounds relpos
+phylolikelihood "{""v_r"":[5,5],""d_r"":[10,10],""j_l"":[10,10],""j_r"":[13,13],""d_l"":[5,5],""v_l"":[0,0]}" "{""IGHD_ex*01"":5,""IGHJ_ex*01"":10,""IGHV_ex*01"":0}"
```

Comment:
all the `phylolikelihood`-related files are made so the full phyloHMM likelihood == standard phylo. likelihood across the MSA. we do this to test our emission calculations via libptpll.